### PR TITLE
part2 exercises complete

### PR DESCRIPTION
**Pull request by mikram5**

### CODE REVIEW

File: src/part_02/Exercise_05.java [hunk 1]
```diff
@@ -12,6 +12,26 @@
...
  */
 public class Exercise_05 {
+    public static void main(String[] args) {
+
+        Scanner scanner = new Scanner(System.in);
+
+        System.out.println("Enter a number: ");
+        int number = scanner.nextInt();
+
+        if (number == 1) { System.out.println("ONE");
```
Comment (roioteromorales) [inline src/part_02/Exercise_05.java]: This `if` construction is not `nested`, they are all at the same level, a nested should be that every `if` is inside the previous one.

File: src/part_02/Exercise_08.java [hunk 2]
```diff
@@ -4,13 +4,25 @@
...
+ * Use while loop to find the number - when the number is found
+ * exit the loop and print the number to the console.
  */
 public class Exercise_08 {
+    public static void main(String[] args) {
 
+        Scanner scanner = new Scanner(System.in);
+
+        System.out.println("Enter a number between 0 and 1,000,000,000");
+        int number = scanner.nextInt();
```
Comment (roioteromorales) [inline src/part_02/Exercise_08.java]: This is good, you are just missing the validation for the input, so you need to check if what the user inputs is between 0 and 1.000.000.

File: src/part_02/Exercise_11.java [hunk 3]
```diff
@@ -18,6 +18,31 @@
...
+
+        System.out.println("Enter an interest rate: ");
+        double irate = scanner.nextDouble();
+
+        System.out.println("Enter the number of years to invest: ");
+        double yearsInvesting = scanner.nextDouble();
+
+
+        for(int i = 0; i < yearsInvesting; i++) {
+            investAmount *= (1 + irate);
```
Comment (roioteromorales) [inline src/part_02/Exercise_11.java]: The formula for investments its not working correctly:
`investAmount *= (1 + irate);`

The correct formula should be something like this:
`investAmount = investAmount + (investAmount * (irate/100));`

File: src/part_02/Exercise_05.java (hunk 1 above)
Comment (mikram5) [reply src/part_02/Exercise_05.java]: made the requested change

File: src/part_02/Exercise_07.java [hunk 4]
```diff
@@ -13,6 +13,30 @@
...
+    }
+
+    public String convertMinutes(int minutes) {
+        int years;
+        int days;
+        int minutesInYear = 525600;
+        int minutesInDay = 1440;
+
+        years = minutes / minutesInYear;
+        days = minutes / minutesInDay;
```
[the remark this answers is not in the document]
Comment (mikram5) [reply src/part_02/Exercise_07.java]: made the requested change

File: src/part_02/Exercise_08.java (hunk 2 above)
Comment (mikram5) [reply src/part_02/Exercise_08.java]: made the requested change

File: src/part_02/Exercise_10.java [hunk 5]
```diff
@@ -3,12 +3,28 @@
...
- *      Write a program that prints out every prime number between 1 and 100
- *
+ * <p>
+ * Write a program that prints out every prime number between 1 and 100
  */
 public class Exercise_10 {
 
+    public static void main(String[] args) {
+
+        for (int num = 1; num <= 100; num++) {
```
[the remark this answers is not in the document]
Comment (mikram5) [reply src/part_02/Exercise_10.java]: made the requested change

File: src/part_02/Exercise_11.java (hunk 3 above)
Comment (mikram5) [reply src/part_02/Exercise_11.java]: made the requested change

**Pull request merged**